package/android/src/main/java/com/shopify/reactnative/skia/PlatformContext.java

```diff
@@ -52,13 +52,13 @@ private void postFrameLoop() {
         Choreographer.FrameCallback frameCallback = new Choreographer.FrameCallback() {
             @Override
             public void doFrame(long frameTimeNanos) {
+                if (_drawLoopActive) {
```

Comment:
I tested these changes and they don't affect the output as far as I was able to measure. To keep this PR clear I'd rather make a separate PR that measure and documents why this change is needed.

I don't think the order is important since this all happens on the main thread? The only race condition I could see was if the frame (notifyDrawLoop) took more than the available time to render then this would cause some stutter - but since it is all on the same thread (main) it would stutter anyways?

Reply:
Yes it will stutter anyways but if the post callback is done after, it will stutter + 1 frame.
When working on this, it felt easy to reproduce this exact behaviour (frame lost vs frame lost + 1) but I will double check

Reply:
I should mention that I found substancial issues/bottlenecks with having a main loop such as this one but decided to now tackle it yet since we may solve these automatically by having a render thread.

Reply:
I agree that this change is not making any difference. I hope it's ok to leave it like this. When trying different setups, it always felt safe to have the callback posted at the beginning.